MAINTAINERS

```diff
@@ -240,6 +256,11 @@ example and you'll be fine".
 		"stevvooe",
 	]
 
+	[Org.Projects]
```

Comment:
Agreed, we can just assume that every project under `github.com/containerd` is under the umbrella of the organization.